src/components/collection/copyModal/ChildrenNavigationTree.tsx

```diff
@@ -0,0 +1,63 @@
+import { useContext } from 'react';
+
+import { Alert, Box, Skeleton } from '@mui/material';
+
+import { ItemType } from '@graasp/sdk';
+import { NavigationElement, RowMenuProps, RowMenus } from '@graasp/ui';
+
+import { useLibraryTranslation } from '../../../config/i18n';
+import LIBRARY from '../../../langs/constants';
+import { QueryClientContext } from '../../QueryClientContext';
+
+interface ChildrenNavigationTreeProps {
+  isDisabled?: RowMenuProps['isDisabled'];
+  selectedId?: string;
+  selectedNavigationItem: NavigationElement;
+  onClick: RowMenuProps['onClick'];
+  onNavigate: RowMenuProps['onNavigate'];
+}
+
+const ChildrenNavigationTree = ({
+  onClick,
+  selectedId,
+  selectedNavigationItem,
+  onNavigate,
+  isDisabled,
+}: ChildrenNavigationTreeProps): JSX.Element => {
+  const { t } = useLibraryTranslation();
+  const { hooks } = useContext(QueryClientContext);
+  const { data: children, isLoading } = hooks.useChildren(
+    selectedNavigationItem.id,
+  );
+  // TODO: use hook's filter when available
+  const folders = children?.filter((f) => f.type === ItemType.FOLDER);
```

Comment:
I think the filter is available now no ? 
```suggestion
  const { data: children, isLoading } = hooks.useChildren(
    selectedNavigationItem.id,
    { types: [ItemType.FOLDER] }
  );
```